### PR TITLE
Create navbar

### PR DESCRIPTION
**Pull request by hsuyuching**
Finished front-end for navbar.

### CODE REVIEW

File: mean-foodRec/src/app/app.component.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<app-navbar></app-navbar>
+<main>
+  <span>Hihihi</span>
+</main>
```
Comment (borkaehw) [inline mean-foodRec/src/app/app.component.html:4]: Remove this part.

File: mean-foodRec/src/index.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
```
Comment (borkaehw) [inline mean-foodRec/src/index.html:2]: Remove empty line

File: mean-foodRec/src/app/app.component.html (hunk 1 above)
Comment (hsuyuching) [reply mean-foodRec/src/app/app.component.html]: done

File: mean-foodRec/src/index.html (hunk 2 above)
Comment (hsuyuching) [reply mean-foodRec/src/index.html]: done

**Pull request merged**